templates/solutions/ai-infrastructure/index.html

```diff
@@ -252,16 +252,16 @@ <h2>Accelerate innovation with certified hardware</h2>
           Get peace of mind with Ubuntu certified hardware from all major OEMs. Choosing validated platforms significantly reduces deployment time and costs for end-customers, in addition to improving reliability and providing bug-fixes faster.
         </p>
         <p class="p-section--shallow">
-          Combined with NVIDIA DGX systems or NVIDIA-Certified systems, choosing an Ubuntu-certified platform provides secure, performant hardware that is guaranteed to deploy quickly and run smoothly.
+          For example, combined with NVIDIA DGX systems or NVIDIA-Certified systems, choosing an Ubuntu-certified platform provides secure, performant hardware that is guaranteed to deploy quickly and run smoothly.
         </p>
         <hr class="is-muted" />
         <p>
-          <a href="/partners/silicon">Explore our silicon programme&nbsp;&rsaquo;</a>
+          <a href="https://ubuntu.com/certified">Explore our certified hardware&nbsp;&rsaquo;</a>
         </p>
-        {{ image(url="https://assets.ubuntu.com/v1/ba3a0335-Canonicla+nvidia.png",
-                alt="",
-                width="536",
-                height="66",
+        {{ image(url="https://assets.ubuntu.com/v1/e301f02e-Canonical%20+%20nvidia.png",
+                alt="Canonical and Nividia logos",
```

Comment:
```suggestion
                alt="Canonical and NVIDIA logos",
```
Looks like a typo in the NVIDIA alt text